data/pools/transactionPool.go

```diff
@@ -244,7 +244,7 @@ func (pool *TransactionPool) computeDeductions(t transactions.SignedTxn) (accoun
 	if err != nil {
 		return algosPendingSpend, fmt.Errorf("TransactionPool.Remember: failed to compute balance - %v", err)
 	}
-	senderBalance, _, _, _, _, err := pool.ledger.BalanceAndStatus(t.Txn.Src())
+	senderBalance, _, _, _, _, _, _, err := pool.ledger.BalanceAndStatus(t.Txn.Src())
```

Comment:
looks like we could refactor the BalanceAndStatus; all these ignored returned values aren't super-readable, and are a good indication of a mismatching interface.

Reply:
( note that it's not really related to this PR; issue was there before )

Reply:
We're planning to deprecate `data/ledger` at some point. Hopefully at that point we can fix up this interface.